### PR TITLE
Fix after seperating scanners

### PR DESCRIPTION
**Pull request by JustinWonjaePark**
## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Refactoring, Maintenance
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: src/fosslight_source/cli.py [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+    elif selected_scanner == 'scanoss':
+        scanned_result = run_scanoss_py(path_to_scan, output_file_name, format, True, write_json_file)
+        create_report_file(scanned_result, license_list, print_matched_text, output_file_name, format)
+    elif selected_scanner == 'all' or selected_scanner == '':
+        success, _result_log["Scan Result"], scanned_result, license_list = run_all_scanners(path_to_scan, output_file_name,
+                                                                                             write_json_file, -1, True,
+                                                                                             print_matched_text, format, True)
+        create_report_file(scanned_result, license_list, print_matched_text, output_file_name, format)
+    else:
+        print("WRONG SCANNER!!!!!!! EXCEPTION NEEDED")
```
Comment (soimkim) [inline src/fosslight_source/cli.py:98]: Let's add sys.exit(1)

File: src/fosslight_source/cli.py [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+            elif opt == "-s":
+                selected_scanner = arg.lower()
+    except Exception:
+        print_help_msg_source()
+
+    timer = TimerThread()
+    timer.setDaemon(True)
+    timer.start()
+
+    print("output_file_name : ", output_file_name)
```
Comment (soimkim) [inline src/fosslight_source/cli.py:77]: Let's remove this print.

File: src/fosslight_source/cli.py [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+    start_time = datetime.now().strftime('%Y%m%d_%H%M%S')
+    success, msg, output_path, output_file, output_extension = check_output_format(output_file_name, format)
+    print("output_file_name : ", output_file_name)
```
Comment (soimkim) [inline src/fosslight_source/cli.py:80]: Let's remove this print

File: src/fosslight_source/cli.py [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+    else:
+        print("WRONG SCANNER!!!!!!! EXCEPTION NEEDED")
+
+
+def create_report_file(scanned_result, license_list, need_license=False, output_file_name="", format=""):
+    _result_log = {}
+    sheet_list = {}
+    _json_ext = ".json"
+
+    start_time = datetime.now().strftime('%Y%m%d_%H%M%S')
```
Comment (soimkim) [inline src/fosslight_source/cli.py:106]: start_time is already defined in #78.

File: src/fosslight_source/cli.py [hunk 5]
```diff
@@ -0,0 +1,177 @@
...
+    except Exception:
+        print_help_msg_source()
+
+    timer = TimerThread()
+    timer.setDaemon(True)
+    timer.start()
+
+    print("output_file_name : ", output_file_name)
+    start_time = datetime.now().strftime('%Y%m%d_%H%M%S')
+    success, msg, output_path, output_file, output_extension = check_output_format(output_file_name, format)
```
Comment (soimkim) [inline src/fosslight_source/cli.py:79]: This is called at create_report_file().

File: src/fosslight_source/run_scanoss.py [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+    logger.info("|---Number of files detected with SCANOSS: " + str(len(scanoss_file_list)))
+
+    if not write_json_file:
+        try:
+            scan_command = "rm " + output_file
+            os.system(scan_command)
+            scan_command = "rm scanner_output.wfp"
+            os.system(scan_command)
+        except Exception as error:
+            logger.warning("Deleting scanoss result failed.:" + str(error))
```
Comment (soimkim) [inline src/fosslight_source/run_scanoss.py:88]: Let's change to logger.debug().
A warning is something that the user needs to check.

File: src/fosslight_source/cli.py (hunk 5 above)
Comment (JustinWonjaePark) [reply src/fosslight_source/cli.py:79]: It still is needed for log initialization.

File: README.md [hunk 7]
```diff
@@ -10,7 +10,7 @@ SPDX-License-Identifier: Apache-2.0
...
+<img src="https://img.shields.io/pypi/l/fosslight_source" alt="FOSSLight Source Scanner is released under the Apache-2.0 License." /> <img src="https://img.shields.io/pypi/v/fosslight_source" alt="Current python package version." /> <img src="https://img.shields.io/pypi/pyversions/fosslight_source" /> [![REUSE status](https://api.reuse.software/badge/github.com/fosslight/fosslight_source_scanner)](https://api.reuse.software/info/github.com/fosslight/fosslight_source_scanner) [![Guide](http://img.shields.io/badge/-doc-blue?style=flat-square&logo=github&link=https://fosslight.org/fosslight-guide-en/scanner/1_source.html)](https://fosslight.org/fosslight-guide-en/scanner/2_source.html)
```
[the remark this answers is not in the document]
Comment (JustinWonjaePark) [reply README.md]: done

File: src/fosslight_source/_parsing_scancode_file_item.py [hunk 8]
```diff
@@ -8,9 +8,13 @@
 import re
 import fosslight_util.constant as constant
 from ._license_matched import MatchedLicense
+from ._scan_item import ScanItem
+from ._scan_item import is_exclude_dir
+from ._scan_item import is_exclude_file
+from ._scan_item import _replace_word
```
[the remark this answers is not in the document]
Comment (JustinWonjaePark) [reply src/fosslight_source/_parsing_scancode_file_item.py]: done

File: src/fosslight_source/_scan_item.py [hunk 9]
```diff
@@ -0,0 +1,149 @@
...
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+import logging
+# import re
+import fosslight_util.constant as constant
+# from ._license_matched import MatchedLicense
+
+logger = logging.getLogger(constant.LOGGER_NAME)
+_replace_word = ["-only", "-old-style", "-or-later", "licenseref-scancode-"]
```
[the remark this answers is not in the document]
Comment (JustinWonjaePark) [reply src/fosslight_source/_scan_item.py]: done

File: src/fosslight_source/cli.py (hunk 1 above)
Comment (JustinWonjaePark) [reply src/fosslight_source/cli.py:96]: done

File: src/fosslight_source/cli.py (hunk 2 above)
Comment (JustinWonjaePark) [reply src/fosslight_source/cli.py]: done

File: src/fosslight_source/cli.py (hunk 3 above)
Comment (JustinWonjaePark) [reply src/fosslight_source/cli.py]: done

File: src/fosslight_source/cli.py (hunk 4 above)
Comment (JustinWonjaePark) [reply src/fosslight_source/cli.py]: done

File: src/fosslight_source/run_scanoss.py (hunk 6 above)
Comment (JustinWonjaePark) [reply src/fosslight_source/run_scanoss.py]: done

**Comment by JustinWonjaePark** [comment]
Replaced with #48

**Pull request closed without merging**